--- ffmpeg-9999-r1.ebuild	2009-07-12 15:08:42.000000000 +0200
+++ ffmpeg-mt-9999-r1.ebuild	2009-07-12 15:09:08.000000000 +0200
@@ -4,12 +4,12 @@
 
 EAPI=2
 
-ESVN_REPO_URI="svn://svn.mplayerhq.hu/ffmpeg/trunk"
+EGIT_REPO_URI="git://gitorious.org/~astrange/ffmpeg/ffmpeg-mt.git"
 
-inherit eutils flag-o-matic multilib toolchain-funcs subversion
+inherit eutils flag-o-matic multilib toolchain-funcs git
 
 DESCRIPTION="Complete solution to record, convert and stream audio and video.
-Includes libavcodec. live svn"
+Includes libavcodec. Live GIT with multithreading."
 HOMEPAGE="http://ffmpeg.org/"
 
 LICENSE="GPL-3"
@@ -61,6 +61,14 @@ DEPEND="${RDEPEND}
 	v4l? ( sys-kernel/linux-headers )
 	v4l2? ( sys-kernel/linux-headers )"
 
+src_unpack() {
+	git_src_unpack
+
+	# libswscale is a submodule - fetch it.
+	cd "${S}"	 
+	git submodule update --init
+}
+
 src_configure() {
 	local myconf="${EXTRA_FFMPEG_CONF}"
 
